projects/demo/src/app/components/csv-entry/csv-entry.component.ts

```diff
@@ -17,6 +18,12 @@ export class CSVEntryComponent {
   public myFile;
   public footerDirection: 'ltr' | 'rtl' = 'ltr';
 
+  @ViewChild(TsCSVEntryComponent, {
+    static: true,
+    read: false,
```

Comment:
Haven't used this `read` option yet....

Reply:
I haven't really needed it before, but it's what hub is using and they are saying this doesn't work, so creating a reproduction they can see